### DISCUSSION TITLE
Translate RGB to Infrared?

### DISCUSSION THREAD

**Discussion by whittenator** [GaParmar/img2img-turbo#83] [Ideas]
Thoughts or ideas on potentially training this to translate RGB imagery to infrared imagery?

Interested in doing some conversions from something like this:
![030051527f66ac4199e987a622bee8db9879a59e](https://github.com/user-attachments/assets/6c83a742-01c2-4d18-8558-cb2b2a72016a)

To this:
![white_hot_030051527f66ac4199e987a622bee8db9879a59e](https://github.com/user-attachments/assets/bcbfb509-64d0-4f7b-a1ad-20ed136b3455)

**Comment by GaParmar** [comment]
Do you have paired data for this task?
